src/ngraph/pattern/matcher.cpp

```diff
@@ -163,7 +185,15 @@ bool pattern::Matcher::match_value(const Output<Node>& pattern_value,
             return false;
         }
     }
-    return pattern_node->match_value(this, pattern_value, graph_value);
+    bool rc = pattern_node->match_value(this, pattern_value, graph_value);
+    if (!rc && is_type<ngraph::op::GetOutputElement>(graph_node))
+    {
```

Comment:
You can do this `GetOutputElement::match_value`

Reply:
I don't understand what you are suggesting. Can you give more detail or a code suggestion?

Reply:
This code is no longer needed since the GOE uses that required it have been removed